src/main/java/bio/terra/model/DRCollectionFiles.java

```diff
@@ -85,6 +92,30 @@ private FileModel getFileModel() {
         try {
             return pathLookup("/", 1);
         } catch (ApiException ex) {
+            // the pathLookup function above is checking the top-level files collection (path = "/")
+            // the code below checks for the case where this returns not found
+            // (status code = 404 not found, message property of response body = File not found)
+            if (ex.getCode() == HttpStatusCodes.STATUS_CODE_NOT_FOUND) {
+
+                try {
+                    // parse the response body to build a JSON object
+                    Map<String, String> errorMap = CommandUtils.getObjectMapper()
+                            .readValue(ex.getMessage(), new TypeReference<Map<String, String>>() {});
```

Comment:
Still looks like you are using the message and not the response body. The comments help!

Reply:
Oh man, that is a git fail on my part, sorry about it and thanks for catching it.

(Probably TMI, but in case you're curious: I was trying to stash it and make a separate commit but then I forgot to un-stash it and commit.)